recipe/spyder-menu-v1.json.bak

```diff
@@ -7,6 +7,12 @@
             "pywscript": "${PYTHON_SCRIPTS}/spyder-script.pyw",
             "workdir": "${PERSONALDIR}/Python Scripts",
             "icon": "${MENU_DIR}/spyder.ico"
+        },
+        {
+            "name": "Spyder __PKG_MAJOR_VER__ Reset",
+            "pyscript": "${PYTHON_SCRIPTS}/spyder-script.py --reset",
+            "workdir": "${USERPROFILE}",
+            "icon": "${MENU_DIR}/spyder.ico"
```

Comment:
Maybe it could be worthy to create a specific icon for the reset? Otherwise the icon to launch Spyder and the icon to reset its preferences would be the same, right?

Reply:
I think that would be a good idea.